### ISSUE TITLE
GLib "g_get_user_runtime_dir undeclared" compilation error

### ISSUE THREAD

**Issue by weberc2** [mattn/go-gtk#144]
Hello, when I try to compile any example (or anything using the go-gtk libraries), I get this output:

```
weberc2:$ go build event/event.go
# github.com/mattn/go-gtk/glib
 error: 'g_get_user_runtime_dir' undeclared (first use in this function)
error: (Each undeclared identifier is reported only once
```

I can reproduce this problem with a program as simple as this:

```
package main
import "github.com/mattn/go-gtk/gtk"

func main () {
    gtk.Init (nil)
    gtk.Main ()
}
```

This is with a branch of the master that I pulled down at roughly noon on 2013/01/28. It was compiled under Ubuntu 10.04.4 with go version 1.0.3.

This is probably related to https://github.com/mattn/go-gtk/issues/140 (I apologize if I should not have created a new issue, but rather posted in issue 140).

Furthermore, I reverted to a previous version of go-gtk that I knew to be working under Ubuntu 12.10 and confirmed that that version worked, so it's not the result of a recent change in go-gtk, but probably something to do with code used in Ubuntu 10.04.
